airflow-core/src/airflow/ui/public/i18n/locales/en/dag.json

```diff
@@ -5,6 +5,26 @@
     "reason": "Reason",
     "title": "Dependencies Blocking Task From Getting Scheduled"
   },
+  "calendar": {
+    "cellSize": "Cell Size",
+    "daily": "Daily",
+    "decreaseSize": "Decrease size",
+    "hourly": "Hourly",
+    "increaseSize": "Increase size",
+    "legend": {
+      "successRateSpectrum": "Success Rate Spectrum",
+      "tooltips": {
+        "failed": "Less than 20% Success",
+        "success100": "100% Success",
+        "successRate20": "20%+ Success Rate",
+        "successRate40": "40%+ Success Rate",
+        "successRate60": "60%+ Success Rate",
+        "successRate80": "80%+ Success Rate"
```

Comment:
I wonder if we want to pass the number into the translation so its easier to change the values in the future without new translations